plugin/gradle.properties

```diff
@@ -1,10 +1,21 @@
-# Project-wide Gradle settings.
+GROUP=com.snap.daggerbrowser
+VERSION_NAME=0.7-SNAPSHOT
```

Comment:
from what I remember SNAPSHOT should not be used in public version names.

Reply:
It should be used in between releases, which is what we want committed to the repo. I've pushed 0.6 as our initial release.